src/core/include/openvino/core/preprocess/preprocess_steps.hpp

```diff
@@ -115,6 +115,20 @@ class OPENVINO_API PreProcessSteps final {
     /// \return Reference to 'this' to allow chaining with other calls in a builder-like manner.
     PreProcessSteps& resize(ResizeAlgorithm alg);
 
+    /// \brief Crop input tensor between begin and end coordinates. Under the hood, inserts `opset8::Slice` operation to
+    /// execution graph. It is recommended to use to together with `ov::preprocess::InputTensorInfo::set_shape` to set
+    /// original input shape before cropping
+    ///
+    /// \param begin Begin indexes for input tensor cropping. Negative values represent counting elements from the end
+    /// of input tensor
+    ///
+    /// \param end End indexes for input tensor cropping. End indexes are exclusive, which means values including end
+    /// edge are not included in the output slice. Negative values represent counting elements from the end of input
```

Comment:
"End indexes are exclusive" is it aligned with ov::Tensor(t, roi) ?

Reply:
Yes, if tensor is `{10,20,30,40,50}` then, ov::Tensor(tensor, Coordinate{1}, {Coordinate{4}) will give you {20,30,40}.
Also it is aligned with python data[begin:end] syntax + with 'Slice' operation